### ISSUE TITLE
DSS_2015_LV95.Siedlungsentwaesserung.Einzugsgebiet: tid AAA EZ20156: value <AAA EZ20156> is not a valid OID

### ISSUE THREAD

**Issue by ponceta** [qgep/QGEP#884]
**Describe the bug**
I get no valid INTERLIS export mainly due to a lot of these error messages :

```
Error: line 6245: DSS_2015_LV95.Siedlungsentwaesserung.Einzugsgebiet: tid AAA EZ20156: value <AAA EZ20156> is not a valid OID
Error: line 6246: DSS_2015_LV95.Siedlungsentwaesserung.Einzugsgebiet: tid AAA EZ20248: value <AAA EZ20248> is not a valid OID
Error: line 6247: DSS_2015_LV95.Siedlungsentwaesserung.Einzugsgebiet: tid AAA EZ20245: value <AAA EZ20245> is not a valid OID
Error: line 6248: DSS_2015_LV95.Siedlungsentwaesserung.Einzugsgebiet: tid AAA EZ20183: value <AAA EZ20183> is not a valid OID
Error: line 6249: DSS_2015_LV95.Siedlungsentwaesserung.Einzugsgebiet: tid AAA EZ20271: value <AAA EZ20271> is not a valid OID
Error: line 6250: DSS_2015_LV95.Siedlungsentwaesserung.Einzugsgebiet: tid AAA EZ20272: value <AAA EZ20272> is not a valid OID
```

`Error: line 6262: DSS_2015_LV95.Siedlungsentwaesserung.Einzugsgebiet: tid AAA EZ21150: duplicate coord at (2750544.406, 1264707.792, NaN)`

`Error: DSS_2015_LV95.Siedlungsentwaesserung.Einzugsgebiet.Perimeter: Overlay coord1 (2748123.771, 1263279.067), coord2 (2748127.132, 1263274.785), tids EZ1677, EZ1677`

If it is a mandatory step toward INTERLIS validity I think it would be good to have such checks before export.

**To Reproduce**
Exact steps to reproduce the behavior:

**Expected behavior**

**Screenshots / data**
If applicable, add screenshots or data to help explain your problem.

**Desktop (please complete the following information):**
 - QGEP plugin version
 - QGEP datamodel version
 - QGEP project file version
 - QGIS Version
 - OS

**Additional context**

**Comment by sjib** [comment]
We should add a `obj_id_check` for that
And correct the testdata - if this is coming from there.
It rather looks like the arbon dataset (old version)

**Comment by ponceta** [comment]
> It rather looks like the arbon dataset (old version)

Yes it is the old Arbon dataset which is still the current demodata for QGEP datamodel releases.

We can document how to correct the data and make the export work (my initial goal), but other entities will face exactly the same issues so it's better to know what to advise when encountering such issues rather than creating a "perfect/virtual" dataset.
